next.config.js

```diff
@@ -23,7 +24,7 @@ const interests = [
 ];
 
 /** @type {import('next').NextConfig} */
-module.exports = {
+module.exports = MillionLint.next()({
```

Comment:
`MillionLint.next()` creates a function that wraps the Next.js config to bootstrap Million Lint. When it builds for production, it will not bootstrap Million Lint.